src/emulator/host/src/app.cpp

```diff
@@ -309,7 +309,8 @@ ExitCode load_app(Ptr<const void> &entry_point, HostState &host, const std::wstr
         auto &imgui_render = host.display.imgui_render;
 
         bool old_imgui_render = imgui_render.load();
-        while (!imgui_render.compare_exchange_weak(old_imgui_render, !old_imgui_render));
+        while (!imgui_render.compare_exchange_weak(old_imgui_render, !old_imgui_render))
+            ;
```

Comment:
Sorry, for here, and i have see, i have set one think no needed here you can remove While, with set just like that so 
>!imgui_render.compare_exchange_weak(old_imgui_render, !old_imgui_render);

Reply:
The while was intentional, it was supposed to lock the current thread until the conditional is satisfied.

Reply:
Oh i see i have think is needed only for loop 
 With press g